Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -360,7 +360,7 @@ void HexEditor::mousemove_event(GUI::MouseEvent& event)
                 return;
 
             m_selection_end = offset;
-            m_position = offset;
+            m_position = (offset + 1 > m_document->size()) ? offset - 1 : offset;
```

Comment:
I can only reproduce the crash in text mode, and selecting the last byte only works with the mouse.

Reply:
Yes, I meant selecting by mouse. It's also reproducible in hex mode. Try selecting (with mouse) last byte and type anything in the view, I get a crash in both text and hex modes.